scripts/lib/mkcloud-common.sh

```diff
@@ -527,7 +527,7 @@ find_fastest_clouddata_server()
         exec cat $cache || exit 100
     fi
     mkdir -p ~/.mkcloud
-    $scripts_lib_dir/find_fastest_server.pl clouddata.nue.suse.com. {,provo-,provo2-}clouddata.cloud.suse.de. | tee $cache
+    $scripts_lib_dir/find_fastest_server.pl clouddata.nue.suse.com. provo-clouddata.cloud.suse.de. | tee $cache
```

Comment:
What happened to `provo2`? Is that not used anymore? Or is at always the slowest? :wink:

Reply:
Having only one Provo host makes things a bit more transparent anyway :smile: Thanks!